lua/lvim/core/terminal.lua

```diff
@@ -146,24 +166,4 @@ M.toggle_log_view = function(logfile)
   log_view:toggle()
 end
 
-M.lazygit_toggle = function()
-  local Terminal = require("toggleterm.terminal").Terminal
-  local lazygit = Terminal:new {
-    cmd = "lazygit",
-    hidden = true,
-    direction = "float",
-    float_opts = {
-      border = "none",
-      width = 100000,
-      height = 100000,
-    },
-    on_open = function(_)
-      vim.cmd "startinsert!"
```

Comment:
we don't need this one anymore?

Reply:
Nope, lazygit is just another terminal in `lvim.builtin.terminal.keybinds` and handled like the others :)